db/migrate/20241203151055_create_account_recovery_request_codes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAccountRecoveryRequestCodes < ActiveRecord::Migration[8.0]
+  def change
+    create_table :account_recovery_request_codes do |t|
+      t.string :hashed_code, null: false
+      t.index :hashed_code, unique: true
```

Comment:
I'm not sure this has to be unique 🤔 these would work the same as a password - we wouldn't want passwords to be unique

Reply:
Yeah, that's true